entity_gym/entity_gym/environment/env_list.py

```diff
@@ -34,6 +36,9 @@ def reset(self, obs_space: ObsSpace) -> ObsBatch:
             self.cls.action_space(),
         )
 
+    def render(self, **kwargs: Mapping[str, Any]) -> np.ndarray:
```

Comment:
I think we probably want a more generic render method that doesn't assume a specific rendering format, but this seems fine for now. We should still add some docs here to describe the encoding used by the output array, and use `numpy.typing.NDArray` for more precise type information (see e.g. train.py).

Reply:
So I'm specifically following the format that add_video allows. I did not end up using the Griddly render tool (which uses ffmpeg and mp4). I think by following tensorboard's format we're probably as close to "generic render method" as we can get.